scripts/notifications.coffee

```diff
@@ -0,0 +1,78 @@
+# Description:
+#   Notifications for botgt
+# Commands:
+#   hubot notify <message> - notify everyone with something
```

Comment:
This may be unnecessary for the MVP, but it could be nice to have a command that prints all available mediums/channels